src/add&delete.js

```diff
@@ -0,0 +1,77 @@
+import isChecked from './updateStatus';
+import editTask from './editTask';
+import clearTasks from './clearTasks'; // eslint-disable-line import/no-cycle
+
+const submitForm = document.querySelector('form');
+const taskInput = document.querySelector('#task-input');
+const todoList = document.querySelector('.list-group');
+
+const printTask = (task) => {
+  const li = document.createElement('li');
+
+  li.classList = 'list-group-item';
+  li.setAttribute('draggable', 'true');
+  li.innerHTML = `
+  <i class="fa fa-grip-vertical drag-drop"></i>
+  <input type="checkbox" class="checkbox">
+  <input type="text" id="${task.index}" value="${task.description}" class="singleTaskInput">
+  <i class="fa fa-trash-alt delete-btn float-end"></i>
+  `;
+
+  document.querySelector('.list-group').appendChild(li);
+
+  if (task.completed === true) {
+    li.children[2].classList.add('checked');
+    li.children[1].checked = true;
+  }
+
+  return li;
+};
+
+const deleteTask = (li, tasks) => {
+  li.children[3].addEventListener('click', () => {
+    for (let i = 0; i < tasks.length; i += 1) {
+      if ((tasks[i].index) === Number(li.children[2].id)) {
+        tasks.splice(i, 1);
+      }
```

Comment:
I believe it is better to use the slice method here because it does not mutate the original array.